roles/bootstrap-os/tasks/bootstrap-archlinux.yml

```diff
@@ -0,0 +1,32 @@
+---
+#  raw: cat /etc/issue.net | grep '{{ bootstrap_versions }}'
+
+- name: Bootstrap | Check if bootstrap is needed
+  raw: which "{{ item }}"
+  register: need_bootstrap
+  failed_when: false
+  changed_when: false
+  with_items:
+    - python2
+    - pip2
+    - ethtool
+    - python2-netaddr
+  environment: {}
+  tags: facts
+
+- name: Bootstrap | Install python 2.x and pip
```

Comment:
Why `python2` and why would you need `pip` installed?

Reply:
It was present on original repository. Removed the uneeded packages.

Reply:
The contrib/vault needed the pip and archlinux does not have this package installed in the default installation.